docs/contributing-to-airbyte/building-new-connector/testing-connectors.md

```diff
@@ -26,7 +26,11 @@ See all the test cases and their description in [Standard Source Tests](standard
 
 ### Setting up standard tests for your connector
 
-Standard tests are typically run from a docker container. The default standard test runner is the File-based Standard Test suite, which gets its name because its inputs are passed as files via Docker volume mounts. This is the simplest way to run the standard test suite: the only requirements are that you place its input files inside your connector's directory, and to pass the paths to those input files as arguments to the Gradle plugin required to invoke it. This is setup by default inside the `build.gradle` file on all connectors generated from templates.
+Standard tests are typically run from a docker container. 
```

Comment:
I think we should remove all references to the v1 test suite into a separate md page (lets say maybe `Standard Tests (Legacy)`) and link to it in this page saying something like `If you are looking for reference documentation for the deprecated first version of test suites, see <link>`. 

The current md file has a little bit of split-brain between v1 and v2. I would rather we start fresh for a v2 document and feel free to copy anything from the v1 document that you like.